source/resampling_with_code.Rmd

```diff
@@ -41,39 +41,164 @@ in probability and statistics.
+the new treatment cured 17 out of 17 patients.  If the new treatment is
+exactly as effective as the standard treatments, at 90% rate of cure, then we
+would expect a lot of the patients to be cured.  But, if it is really true
+that each patient has a 90% chance of cure, how likely is it that all 17
+patients would be cured?  It seems like something that could happen, but how
+likely is it? In other words, is the result that the researchers saw something
+special that warrants further investigation, or is it something that could reasonably have come about by chance?
+
+This is exactly the kind of question we ask in statistics — is this result
+*surprising* or *unusual*, given that we know that there is an element of
+chance?
+
+First we come up with a *model* of the world we expect, when the treatment is
+90% effective.  This model has to allow for *chance* because each patient has
+a 90% *chance* of cure.
+
+It is often useful to think how we would make a physical model of chance in
+this world.  For example, to simulate whether a given patient is cured or not
+by a 90% effective treatment, we could throw a ten sided die and record the
+result.  We could say, arbitrarily, that a result of 0 means "not cured", and
+all the numbers 1 through 9 mean "cured" (typical 10-sided dice have sides
+numbered 0 through 9).
+
+We could roll 17 dice to simulate one "trial" in this random world.  For each
+trial, we record the number of dice that show numbers 1 through 9 (and not
+0).  This will be a number between 0 and 17, and it is the number of patients
+"cured" in our simulated trial.
+
+Here is the result of one such trial we did with a set of 17 10-sided dice we happened to have near to hand:
+
+:::{.callout-note}
+We need a photo of 17 10-sided dice here
+:::
 
-## A treatment for Burkitt's lymphoma
+In this case there is one 0, and the rest are from 1 through 9, so there were 16 out of 17 "patients" who were "cured" in this trial.
 
-In 2006, researchers came up with a new treatment for Burkitt's
-lymphoma—a cancer of the lymphatic system.  In developed countries,
-the cure rate for Burkitt's lymphoma is about 90%.  The new treatment
-was applied to 17 patients, and all 17 patients were "cured"
-(were progression-free after two years).
+<!---
+Adapt above to picture.
+-->
 
-The first question that comes to mind is: how likely is that to happen
-by chance?  In other words, is what we saw something special that
-warrants further investigation, or is it pretty common for 17
-randomly chosen patients to all be cured using a 90% effective
-treatment?
+We can use the computer to do something very similar, by asking the computer
+for 17 random whole numbers from 0 through 9.
 
-Before we start, we again import NumPy and get its random number generator:
+:::{.callout-note}
+## Whole numbers
 
-```{python}
+A whole number is a number that is not negative, and does not have fractional
```

Comment:
Whole numbers are a woolly concept.  Maybe rather go with something explicit like integers from 0 through 9.

Reply:
OK to resolve this one?